site/pages/docs/versions/dwnld-en.hbs

```diff
@@ -80,8 +102,8 @@
 				<tr>
 					<td>Canada.ca theme (GCWeb)</td>
 					<td>v4.0.32</td>
-					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.0.zip">Download<span class="wb-inv"> - v6.0 source code - Canada.ca theme</span> (theme only)</a></td>
-					<td><a href="https://github.com/wet-boew/GCWeb/releases/download/v6.0/themes-dist-6.0-gcweb.zip">Download<span class="wb-inv"> - v6.0 production files - Canada.ca theme</span></a></td>
+					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.0.zip">Download<span class="wb-inv"> - v6.1 source code - Canada.ca theme</span> (theme only)</a></td>
```

Comment:
```suggestion
					<td><a href="https://github.com/wet-boew/GCWeb/archive/v6.1.zip">Download<span class="wb-inv"> - v6.1 source code - Canada.ca theme</span> (theme only)</a></td>
```

Reply:
It is under 6.0. Does this still have to be 6.1?